pyrevitlib/pyrevit/forms/__init__.py

```diff
@@ -1169,6 +1174,24 @@ def clear_search(self, sender, args):    #pylint: disable=W0613
         self.search_tb.Clear()
         self.search_tb.Focus()
 
+    @classmethod
+    def show(cls, context,  #pylint: disable=W0221
+             title='User Input',
+             width=DEFAULT_INPUTWINDOW_WIDTH,
+             height=DEFAULT_INPUTWINDOW_HEIGHT, **kwargs):
+        """Show user input window.
+
+        Args:
+            context (any): window context element(s)
+        """
+        dlg = cls(context, title, width, height, **kwargs)
+        dlg.ShowDialog()
+
+        if dlg.exitscript and not dlg.response:
+            sys.exit()
```

Comment:
it makes no sense to add the exiscript to the class if it is only used in the class method

```suggestion
    @classmethod
    def show(
        cls,
        context,
        title='User Input',
        width=DEFAULT_INPUTWINDOW_WIDTH,
        height=DEFAULT_INPUTWINDOW_HEIGHT,
        exitscript=False
        **kwargs
    ):
        """Show user input window.

        Args:
            context (any): window context element(s)
            width: ...
            height: ...
            exitscript: exit the script if the user cancels the dialog. Defaults to False
        """
        dlg = cls(context, title, width, height, **kwargs)
        dlg.ShowDialog()

        if exitscript and not dlg.response:
            sys.exit()
```